tasks/bootstrap-services-and-os-settings.yml

```diff
@@ -14,6 +14,15 @@
   tags:
     - set-hostname
 
+- name: Add hostname entry to /etc/hosts
+  lineinfile:
+    path: /etc/hosts
+    line: '127.0.0.1    {{ hostname }}'
```

Comment:
Is widely recommended to have `127.0.0.1 localhost {{ hostname }}` instead of just the domain name :)
can we put it @pibism ? :)

Reply:
@speedlight, added in 69a9233ffa7eafcb48f863ab44ce3450ae0120f6

Reply:
Thanks @pibism !